docs/node-mixin/rules/rules.libsonnet

```diff
@@ -0,0 +1,116 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'node-exporter.rules',
+        rules: [
+          {
+            // This rule gives the number of CPUs per node.
+            record: 'instance:node_num_cpu:sum',
+            expr: |||
+              count without (cpu) (
+                sum without (mode) (
+                  node_cpu_seconds_total{%(nodeExporterSelector)s}
+                )
+              )
+            ||| % $._config,
+          },
+          {
+            // CPU utilisation is % CPU is not idle.
+            record: 'instance:node_cpu_utilisation:avg_rate1m',
```

Comment:
Hmm, I think in this case this is just a :rate1m.

Reply:
It's an average over all cores of an instance (`cpu` label is averaged away). Isn't that exactly the case where you want to keep the operation in the name? It's for example different from the summed up utilisation of all cores of an instance.

Reply:
Given you've changed the metric name, I think that this number represents the number of seconds non-idle per second and so the avg is only an implementation detail.

Reply:
Could still be seen as the _sum_ of non-idle seconds per second (as I have just learned that `sum` is the default operations. But yeah, "utilisation" probably implies some ratio from 0 to 1. I don't feel that strongly, just tried to be strict. But a shorter name has its merits, too. Will remove the avg then.